libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -172,6 +172,19 @@ private List<VcStoreItem> getSuccessfulVCStoreItems(List<VcStoreItem> vcStoreIte
         return successfulVCStoreItems;
     }
 
+    public Optional<Boolean> getVCSuccessStatus(String userId, String criIssuer)
+            throws ParseException {
+        List<VcStoreItem> vcStoreItems = getVcStoreItems(userId);
+        for (VcStoreItem vcStoreItem : vcStoreItems) {
+            SignedJWT vc = SignedJWT.parse(vcStoreItem.getCredential());
+            if (vc.getJWTClaimsSet().getIssuer().equals(criIssuer)) {
+                return Optional.of(
+                        VcHelper.isSuccessfulVc(vc, VcHelper.getNonEvidenceCredentialIssuers()));
```

Comment:
We have a method `VcHelper.isSuccessfulVc(SignedJWT vc)` which doesn't require you to pass in a list of non evidence CRIs. It actually uses the same method you're using here. Might be cleaner to just use that.

Reply:
Done.